### ISSUE TITLE
Question about multi-task

### ISSUE THREAD

**Issue by rOtking** [dvlab-research/DeepVision3D#5]
I have a question,  does the model can support multi-task? Or the architecture can support multi-task, but you can only choose one specific task when you train.

**Comment by tomztyang** [comment]
Yes. You can train one model on multi-task simultaneously with accordingly set query positions.

**Comment by rOtking** [comment]
@tomztyang Thank you for your reply. And i want to know, does one query can be used for multi-task with different head? Or the querys are grouped, for example, some querys are used for detection, some querys are used for segmentation?

**Comment by tomztyang** [comment]
In my opinion, a query represents a position. You can use features of this position for detection, or for segmentation, or for both simultaneously, which depend on the head. If you wish to use the features for multi-tasking, just add multi-task head after obtaining the feature of this position. That's my though.
